solutions/gold/csa-MountainTime.mdx

```diff
@@ -0,0 +1,149 @@
+---
+id: csa-MountainTime
+source: CSA
+title: Mountain Time
+author: Ryan Chou
+---
+<Spoiler title="Hint 1">
+Instead of thinking of the problem as a grid, think of it as a graph, where we're adding one cell at a time.
+
+In what order should we process these cells?
+</Spoiler>
+<Spoiler title="Hint 1 (answer)">
+We'll process them in decreasing order of height.
+</Spoiler>
+
+
+<Spoiler title="Hint 2">
+Whenever we process a cell, we'll want to merge it with all of its neighbors that have already been added (taller than it).
+
+What data structure should we use such that we can efficiently merge two components?
+</Spoiler>
+
+
+<Spoiler title="Hint 2 (answer)">
```

Comment:
maybe "Answer to Hint [x]" instead

Reply:
wait we wrote it this way for everything else tho >.<

Reply:
i remember ctrl f + replacing them some time ago